openapi-codegen/core/src/main/scala/sttp/tapir/codegen/openapi/models/OpenapiComponent.scala

```diff
@@ -5,11 +5,14 @@ case class OpenapiComponent(schemas: Map[String, OpenapiSchemaType])
 object OpenapiComponent {
   import io.circe._
 
-  implicit val OpenapiComponentDecoder: Decoder[OpenapiComponent] = { (c: HCursor) =>
+  implicit val OpenapiComponentDecoder: Decoder[Option[OpenapiComponent]] = { (c: HCursor) =>
```

Comment:
**question:** Shouldn't it be enough to keep the `OpenapiComponent` decoder? The `Option[]` wrapper should be then derived by circe?

Reply:
I just tried it with this file unchanged, and tests pass :)